scripts/apps/desks/directives/DeskSelect.js

```diff
@@ -29,6 +29,14 @@ export function DeskSelect(Keys, _) {
 
             scope.filter = '';
 
+            preferencesService.get('desks:preferred').then(function(result) {
+                scope.preferredDesks = result;
+            });
+
+            scope.deskPreferenceOrdering = function(desk) {
```

Comment:
Do you mind documenting this function please? As well as the `scope.desks` you've added further down in `UserPreferencesDirective`. You'll have to make sure they are displayed correctly in `grunt docs`. See our [contribution guidelines](https://github.com/superdesk/superdesk-client-core/blob/master/CONTRIBUTING.md) for more information.

For them to display on that page, you'll also have to document their host parents, which are `DeskSelect` (remember to use the name of the directive, and not _DeskSelect_, in the documentation), as well as `UserPreferencesDirective`.

I would really appreciate it if you could do this. We have to try and add more and more documentation and for this to happen we could really use some help :-)